pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -71,6 +73,9 @@
 # > the experience of Firefox users, you won't have any issues using the
 # > official branding.
 , enableOfficialBranding ? true
+
+# We disable adding "-g" for easier linking on 32bit platform.
```

Comment:
Do we disable "adding `-g`" on all platforms, so that linking is easier on 32bit platform? Or do we disable "adding `-g`" only on 32bit platform, for easier linking there? If the latter (which I assume from the code below), maybe phrase it as:
```suggestion
# On 32bit platform, we disable adding "-g" for easier linking.
```
so that it's clearer, what "on 32bit platform refers to.

Also, is "32bit" just one platform or a category of (potentially or actually) _several_ platforms? If the latter, put it in plural:
```suggestion
# On 32bit platforms, we disable adding "-g" for easier linking.
```

Reply:
About 32bit platforms, I notice that we actually mark `firefox` broken when buildPlatform is 32bit, in,
https://github.com/NixOS/nixpkgs/blob/07534dee7f45a9945123e7bd1580b6c22bb4cdc5/pkgs/applications/networking/browsers/firefox/packages.nix#L22

Maybe we could remove these special cases in `common.nix`, but I'm not sure about testing 32bit builds. So I just keep them currently.